jwst/wfs_combine/wfs_combine_step.py

```diff
@@ -18,7 +18,7 @@ class WfsCombineStep(Step):
 
     def process(self, input_table):
 
-        asn_table = json.load(open(input_table, 'r'))
+        asn_table = self.load_as_level2_asn(input_table)
```

Comment:
The wfs_combine_step is a level-3 step that combines multiple (usually a pair of) dithered WFS&C images, so the ASN file should be composed as a level3 ASN, not level2.

Reply:
Thanks for catching that @hbushouse.  The regression test did pass, but perhaps it is a special case.  Will change this in new PR.